### ISSUE TITLE
Import @Route is never used

### ISSUE THREAD

**Issue by ragusa87** [Haehnchen/idea-php-annotation-plugin#75]
I got a Symfony controller using the "Route" annotation.
Since my upgrade from PhpStorm 2016.3 to 2016.3.1, I got a warning telling me that the use statement for Route is never used.

`Symfony Plugin v 0.12.131`
`PHP Annotations v 4.2`
`Phpstorm v. 2016.3.1`

```php
namespace MyIssueNameSpace;
use Symfony\Component\Routing\Annotation\Route;
use Symfony\Bundle\FrameworkBundle\Controller\Controller;

/**
 * Class ContestController
 * @Route("/contest")
 */
class ContestController extends Controller
{
// ..
}
```
![image](https://cloud.githubusercontent.com/assets/1695207/21011125/24fb1462-bd50-11e6-94bd-76dc51abe241.png)

I do not know if i am posting this issue to the right project.
Thanks for your help !

**Comment by arkste** [comment]
Can confirm. PHP Annotations Plugin is working on `2016.3`, but not on `2016.3.1`.

**Comment by tacman** [comment]
Funny, looks like this happened back in 2013 too: https://youtrack.jetbrains.com/issue/WI-14448

Not working for me, 2016.3.1

The Doctrine annotations work as expected, but the Symfony ones are flagged as unused (e.g. @Groups).

**Comment by Basster** [comment]
Can confirm for all FrameworkExtraBundle\Configuration annotations.

```
PhpStorm 2016.3.1
Build #PS-163.9735.1, built on December 6, 2016

Symfony Plugin: 0.12.131
PHP Annotations: 4.2
```

**Comment by Scifa** [comment]
Confirmed bug :(

https://youtrack.jetbrains.com/issue/WI-34281

**Comment by ragusa87** [comment]
@Scifa Thanks for your feedback. As it's a PhpStorm issue (https://youtrack.jetbrains.com/issue/WI-34285), I will close this one.

**Comment by podhy** [comment]
already fixed in phpstorm 2016.3.2 EAP
